source/TextWrangler/textwrangler.applescript

```diff
@@ -0,0 +1,4 @@
+tell application "TextWrangler"
+	activate
+	make new text document with properties {contents:"{popclip text}"}
```

Comment:
I wanted to have the selected text appear in a new document, **not** in a new window.